### ISSUE TITLE
text not working, and alot of bugs

### ISSUE THREAD

**Issue by mustafaGFX** [photopea/photopea#2753]
text not work
and when I try to export an SVG file photopea render in so weird
and when I try to import an SVG file the same problem happens

![Screenshot (77) (1)](https://user-images.githubusercontent.com/70863817/104459368-50532580-55b5-11eb-987a-6c01206c797c.png)
![Screenshot (77)](https://user-images.githubusercontent.com/70863817/104459375-51845280-55b5-11eb-8843-3cb9b1e286b8.png)

**Comment by mustafaGFX** [comment]
the text is more important, please try to solve this problem as soon as you can

**Comment by photopea** [comment]
Hi, the text was not working yesterday afternoon for about 12 minutes. Did you open Photopea during that time? Can you close Photopea and open it again?

Could you send me your SVG file, which looks in Photopea differently than in another program?

**Comment by mustafaGFX** [comment]
> Hi, the text was not working yesterday afternoon for about 12 minutes. Did you open Photopea during that time? Can you close Photopea and open it again?

the text tool still not working, it depends on the size of the text or the font, some times it works well but when I resize the text or change the font it disappears.
also when I edit the text, make it bold, rotate it, or even when I make a little change in it

> Could you send me your SVG file, which looks in Photopea differently than in another program?
almost all files that have path fill rule

**Comment by mustafaGFX** [comment]
I cant export SVG from these files : 
[vector logos.zip](https://github.com/photopea/photopea/files/5809271/vector.logos.zip)

**Comment by mustafaGFX** [comment]
I think file number 2 isn't exportable in SVG format because of the vector mask, but file number one is so weird

**Comment by photopea** [comment]
Text not working: Could you describe steps, how can I reproduce that issue? I opened "pea.psd" and text layers were editable.

**Comment by mustafaGFX** [comment]
wow
I have just tried to edit the text that had the bug 10 minutes ago and it has been fixed, don't know-how.
maybe because my device was exhausted

**Comment by mustafaGFX** [comment]
the latest screen that shown in the other tab is shown in the PSD export window

![Screenshot (78)](https://user-images.githubusercontent.com/70863817/104470765-bd20ec80-55c2-11eb-995e-d3ebb9642b22.png)

**Comment by photopea** [comment]
I noticed, that the gradients in your "logo design 1.psd" look a bit different in Photoshop than in Photopea. I fixed it (they look the same now). You might need to adjust your gradients to make them look as before.

I improved the SVG export: The SVG of both files look 100% the same as the document in Photopea.

I improved the PDF export. But there is a problem -the PDF format does not have a "feather" (blur) of vector shapes. I recommend rasterizing the whole document in such cases ("rasterize" checkbox in PDF export).
